docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
+
+### Setup the certificate
```

Comment:
I have a minikube cluster set up here and no self-signed certificate was required. Just `minikube start; helm init; draft init`. I feel like this is "extra" documentation that depends on what application you're installing. None of the examples set up tls by default in the ingress routes. I would cut this entire section out IMO.